src/main/java/jenkins/plugins/office365connector/Webhook.java

```diff
@@ -193,6 +199,43 @@ public String getDisplayName() {
             return "Webhook";
         }
 
+        public String getIp() {
+            return Util.fixNull(ip);
+        }
+
+        @DataBoundSetter
+        public void setIp(String ip) {
+            this.ip = Util.fixNull(ip);
+        }
+
+        public String getUsername() {
+            return Util.fixNull(username);
+        }
+
+        @DataBoundSetter
+        public void setUsername(String username) {
+            this.username = Util.fixNull(username);
+        }
+
+        public String getPassword() {
+            return Util.fixNull(password);
+        }
+
+        @DataBoundSetter
+        public void setPassword(String password) {
+            this.password = Util.fixNull(password);
+        }
+
+        @Nonnull
+        public String getPort() {
```

Comment:
Reorder and put port together with ip/address

Reply:
done